AutoRest/Generators/CSharp/CSharp/TemplateModels/ParameterGroupTemplateModel.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Microsoft.Rest.Generator.ClientModel;
+using Microsoft.Rest.Generator.Utilities;
+
+namespace Microsoft.Rest.Generator.CSharp
+{
+    public class ParameterGroupTemplateModel
+    {
+        private readonly Parameter groupParameter;
+
+        public ParameterGroupTemplateModel(ServiceClient serviceClient, Parameter groupParameter)
+        {
+            if (groupParameter == null)
+            {
+                throw new ArgumentNullException("groupParameter");
+            }
+
+            this.LoadFrom(serviceClient);
+            this.groupParameter = groupParameter;
+            
+            CompositeType compositeType = this.groupParameter.Type as CompositeType;
+
+            this.GroupedParameters = compositeType.Properties;
+            this.Documentation = compositeType.Documentation;
+        }
+
+        public string ParameterGroupName
+        {
+            get { return this.groupParameter.Type.Name; }
```

Comment:
@yugangw-msft This field is actually never used anywhere, since the "name" of the parameterGroup is the name of the parameter on the method.  I've removed this property and just left the other property `ParameterGroupType`